schemas/com.delighted/survey_response_updated/jsonschema/1-0-0

```diff
@@ -0,0 +1,159 @@
+                    "description": "NPS score",
+                    "minimum": 0,
+                    "maximum": 100
+                },
+                "comment": {
+                    "type": ["string", "null"],
+                    "maxLength": 512,
+                    "description": "Survey comment"
+                },
+                "permalink": {
+                    "type": "string",
+                    "maxLength": 255,
+                    "description": "Permalink to the response"
+                },
+                "created_at": {
+                    "type": "integer",
+                    "minimum": 0,
+                    "maximum": 2147483647,
+                    "description": "Response created at (in epoch seconds)"
+                },
+                "updated_at": {
+                    "type": "integer",
+                    "minimum": 0,
+                    "maximum": 2147483647,
+                    "description": "Response updated at (in epoch seconds)"
+                },
+                "person_properties": {
+                    "type": ["object", "null"],
+                    "description": "Custom user properites",
+                    "properties": {
+                        "delighted_source": {
+                            "type": ["string", "null"],
+                            "description": "Visitor source e.g., Web",
+                            "maxLength": 50
+                        },
+                        "Page": {
+                            "type": ["string", "null"],
+                            "maxLength": 255
+                        },
+                        "Page URL": {
+                            "type": ["string", "null"],
+                            "maxLength": 512
+                        },
+                        "Referrer URL": {
+                            "type": ["string", "null"],
+                            "maxLength": 512
+                        },
+                        "User Agent": {
+                            "type": ["string", "null"],
+                            "maxLength": 256
+                        },
+                        "IP Address": {
+                            "type": ["string", "null"],
+                            "maxLength": 45
+                        },
+                        "Device Type": {
+                            "type": ["string", "null"]
+                        },
+                        "Operating System": {
+                            "type": ["string", "null"]
+                        },
+                        "Browser": {
+                            "type": ["string", "null"]
+                        },
+                        "Country": {
+                            "type": ["string", "null"]
+                        },
+                        "user_id": {
+                            "type": ["string", "null"],
+                            "maxLength": 100,
```

Comment:
Should be 255? https://github.com/snowplow/snowplow/blob/master/4-storage/redshift-storage/sql/atomic-def.sql#L41